include/wx/stc/completionitem.h

```diff
@@ -0,0 +1,25 @@
+#ifndef _WX_STC_COMPLETIONITEM_H_
+#define _WX_STC_COMPLETIONITEM_H_
+
+#include "wx/string.h"
+#include "wx/clntdata.h"
+
+class WXDLLIMPEXP_STC wxCompletionItem
+{
+public:
+    wxCompletionItem(const wxString &name, int icon, const wxString &doc = wxString(), wxClientData *data = NULL);
+    virtual ~wxCompletionItem();
+
+    const wxString &GetName();
```

Comment:
This is a global style comment: please put `&` near the type, not the identifier. I.e.
```suggestion
    const wxString& GetName();
```